### PR TITLE
DOC-8190: PR #104449 - sql: add default_value and is_overridden to cluster settings

### PR DESCRIPTION
**Pull request by florence-crl**
Addresses DOC-8190

Updated SHOW CLUSTER SETTING page, added `default_value` and `origin` columns.

### CODE REVIEW

**Comment by florence-crl** [comment]
Hi @maryliag , just to clarify, I noticed that if you change the default value a cluster setting and then you change the value **back** to the default, `origin` will remain `override`:

```
root@localhost:26257/test> select * from crdb_internal.cluster_settings where variable = 'server.cpu_profile.duration';
           variable           | value | type | public |                   description                    | default_value |  origin  |             key
------------------------------+-------+------+--------+--------------------------------------------------+---------------+----------+------------------------------
  server.cpu_profile.duration | 10s   | d    |   f    | the duration for how long a cpu profile is taken | 10s           | override | server.cpu_profile.duration
(1 row)
```

**Comment by maryliag** [comment]
that is correct. That is for the case where we might change the default value, so you would still want to keep the value set. If you reset, then is should show default, but if you just set as the same value, it will be override.

File: src/current/v23.2/show-cluster-setting.md [hunk 1]
```diff
@@ -74,7 +74,9 @@ Field | Description
 `value` | The value of the cluster setting.
 `setting_type` | The type of the cluster setting.<br>Possible values for `setting_type` include:<ul><li>`b` (`true` or `false`)</li><li>`z` (size, in bytes)</li><li>`d` (duration)</li><li>`e` (one of a set of possible values)</li><li>`f` (floating-point value)</li><li>`i` (integer)</li><li>`s` (string)</li></ul>
 `description` | A brief description of the cluster setting, including possible values.
-`public` | `true` if the cluster setting is public.<br>This field is only included only if all cluster settings are displayed.
+`public` | `true` if the cluster setting is public.<br>This field is only included if all cluster settings are displayed.
+`default_value` | The default value of the cluster setting.
+`origin` | The origin of the current value of the cluster setting.<br>Possible values for `origin` include:<ul><li>`default` (The current value has not been changed from the default value.)</li><li>`override` (The current value has been changed from the default value.)</li></ul>
```
Comment (rmloveland) [inline src/current/v23.2/show-cluster-setting.md:79]: SO EXCITED FOR US TO HAVE THESE FIELDS :-)

**Review by rmloveland — APPROVED** [review]
LGTM!!!

**Pull request merged**